crates/uv/src/commands/tool/run.rs

```diff
@@ -267,71 +265,65 @@ fn get_entrypoints(
 /// Display a list of tools that provide the executable.
 ///
 /// If there is no package providing the executable, we will display a message to how to install a package.
-async fn list_available_tools(
+async fn show_help(
     invocation_source: ToolRunCommand,
     cache: &Cache,
     printer: Printer,
 ) -> anyhow::Result<()> {
+    let help = format!(
+        "See `{}` for more information.",
+        format!("{invocation_source} --help").bold()
+    );
+
     writeln!(
         printer.stdout(),
-        "Provide a command to invoke with `{invocation_source} <command>` \
-            or `{invocation_source} --from <package> <command>`.\n"
+        "Provide a command to run with `{}`.\n",
+        format!("{invocation_source} <command>").bold()
     )?;
 
     let installed_tools = InstalledTools::from_settings()?;
-    let no_tools_installed_msg =
-        "No tools installed. See `uv tool install --help` for more information.";
```

Comment:
After playing with this a bit more, I was worried users would feel like they had to use `uv tool install` before they could use `uv tool run` which isn't the case so I dropped this message.

Reply:
👍